Fix #187: Respect DataReader limit in paginators (#191)
yiisoft/data: README.md

- The total number of pages is not available
- Can't get to specific page, only "previous" and "next"
- Data cannot be unordered

> - The limit set in the data reader is ignored

Reviewer: ```suggestion
- The limit set in the data reader leads to throw exception
```
